README.rst

```diff
@@ -148,9 +148,27 @@ The recipe supports the following options:
 **flake8**
     If set to True, run Flake8 code analysis. Default is ``True``.
 
+**flake8-extensions**
+    Flake8 now takes advantage of ``flake8`` extension system. Default is None.
+    Example to supercharge with some extensions.
+    ::
+
+        [code-analysis]
+        recipe = plone.recipe.codeanalysis
+        flake8 = True
+        flake8-extension =
+            flake8-blind-except
+            flake8-coding
+            flake8-debugger
+            flake8-quotes
+            pep8-naming
+
+    All through flake8 extensions raised validation errors may be suppressed
+    using the ``flake8-ignore`` option.
+
 **flake8-ignore**
     Skip errors or warnings. See `Flake8 documentation`_ for error codes.
-    Default is none.
+    Default is None.
```

Comment:
done in https://github.com/plone/plone.recipe.codeanalysis/commit/07173705ac82866dfec42499ba5536b141febd63